cmd/minimock/minimock.go

```diff
@@ -383,6 +384,7 @@ func processArgs(args []string, stdout, stderr io.Writer) (*options, error) {
 
 	fs.BoolVar(&opts.noGenerate, "g", false, "don't put go:generate instruction into the generated code")
 	fs.StringVar(&opts.suffix, "s", "_mock_test.go", "mock file suffix")
+	fs.BoolVar(&opts.generateFromAliases, "from-aliases", false, "generate mocks from interface aliases as well")
```

Comment:
discussable if we should ad this feature as an option or make it as default behaviour

Reply:
Hey, Simon

I think this should be default behavior.

Reply:
fixed